.github/workflows/test.yml

```
@@ -53,7 +53,7 @@ jobs:
       matrix:
         python-version: ["3.9", "3.10", "3.11", "3.12"]
         airflow-version: ["2.4", "2.5", "2.6", "2.7", "2.8", "2.9", "2.10", "2.11", "3.0"]
-        dbt-version: ["1.9"]
+        dbt-version: ["1.10"]
```

```suggestion
        dbt-version: ["1.9", "1.10"]
```
Or was that intentional?

---

It was intentional! In theory, our unit tests should not be using dbt - it is being installed only in our integration tests